docs/platforms/javascript/guides/react/features/component-tracking.mdx

```diff
@@ -6,6 +6,18 @@ description: "Learn how Sentry's React SDK allows you to monitor the rendering p
 
 Sentry helps you track your React components and unlock additional insights in your application. You can set it up to track React component names as well as monitor the performance of your React components with component tracking.
 
+So instead of looking at selectors like this:
+
+```html
+div.row > div.col-sm-8.col-md-12.col-lg-8.col-xl-9 > div#main > h3#ignore-exceptions > a[aria-label="common options permalink"]
```

Comment:
might be more relevant to show a selector like this

`button.en302zp1.app-191aavw.e16hd6vm2[role="button"][data-test-id="common-options"]`

Reply:
Thanks!